data/p/dx-by-axolo-to-improve-developer-experience.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "A free tool to make developer feedback effortless and insightful: DX by Axolo"
+date: "2023-02-28"
+tags: ["developer experience"]
+draft: false
+summary: "DX by Axolo is designed to be a free tool to help developers share what they think is working great, and what need to be improved at work."
+image: "/blog/static/images/dx/dx-blog-banner.png"
+---
+
+<ImageContainer
+  alt="sweaty-hands"
+  src="/blog/static/images/dx/dx-meta-banner.png"
+  classNameDiv="mx-10 lg:mx-28"
+  classNameImage=""
+  width={2400}
+  height={1260}
+/>
+
+We've been working with developers for more than 2 years now. We started Axolo as a way to help developer reduce code review fatigue and give reviewers a tool to help them review faster.
+
+Everyday, we're learning more about what developers think is a great environment to work, and today we want to give them a voice to improve how their company shape their work environment with our latest free product: DX by Axolo.
+
+## <span className="rainbow-button block w-fit pr-3 text-2xl font-bold hover:cursor-auto">Give your developers a voice</span>
+
+Engineering managers lack key insights into their team's work environment. The main three points we want to help managers with are:
+
+- How can I identify our team's bottleneck?
+- How to make sure everyone is aligned?
```

Comment:
tu penses quoi de mettre ca en 2e question?
```suggestion
- Are we improving our software development process?
```

Reply:
si tu pref je mets ça, mais en vrai je sais pas ce qui est le plus important pour engineering manager. garder tout le monde motivé et aligné, ou améliorer les process.

ils ont un role de manager, et souvent ils savent pas faire ça. C'est pour ça qu'athenian tape aussi sur "developing leadership" [](edited-with-Axolo)

Reply:
oui c’est juste que j’ai l’impression qu’on mieux a “are we improving our processes” que “how to make sure we are aligned” [](edited-with-Axolo)